### DISCUSSION TITLE
iOS App Server URL

### DISCUSSION THREAD

**Discussion by cmlaird** [chuot/rdio-scanner#201] [Q&A]
I can't seem to figure out which URL to put in the iOS app. I've done ifconfig on the pi to get the IP address but none of the one's I'm seeing work. When I hover over the WiFi symbol in the menu car, I input the IP address and port shown but it still doesn't connect. Where do I need to look to get it? Thanks in advance!

**Comment by chuot** [comment]
It's exactly the same URL as you would use to access the webapp from the Internet.

I would start by confirming you are able to access the webapp from the Internet and making sure it's fully functionnal. Then use that same URL for the native app.
